shared/chat/conversation/bottom-banner/index.tsx

```diff
@@ -72,10 +77,72 @@ const BrokenTrackerBanner = ({users, onClick}: BrokenTrackerProps) =>
     </BannerBox>
   )
 
-const InviteBanner = ({users}: InviteProps) => (
-  <BannerBox color={globalColors.blue}>
-    <BannerText>Your messages to {users.join(' & ')} will unlock when they join Keybase.</BannerText>
-  </BannerBox>
-)
+const InviteBanner = ({users, openSMS, openShareSheet}: InviteProps) => {
+  if (!flags.sbsContacts) {
+    return (
+      <BannerBox color={globalColors.blue}>
+        <BannerText>Your messages to {users.join(' & ')} will unlock when they join Keybase.</BannerText>
+      </BannerBox>
+    )
+  }
+
+  // On mobile, single recipient, a phone number
+  if (isMobile && users.length === 1 && users[0].endsWith('@phone')) {
+    return (
+      <BannerBox color={globalColors.blue}>
+        <BannerText>Last step: summon Firstname Lastman!</BannerText>
+        <Button
+          label="Send install link"
+          onClick={() => openSMS(users[0].slice(0, -6))}
+          mode="Secondary"
+          style={{
+            marginTop: globalMargins.xxtiny,
```

Comment:
In general we prefer to use wrapping `Box2`s with `gap`s instead of setting all these margins, it's one of the uses it was made for.